docs/user/adding-new-metrics.md

```diff
@@ -66,33 +73,70 @@ The `lifetime` parameter of a metric defines when it will be reset. There are th
 - `user`: The metric is part of the user's profile.
   This should be used for things that change only when the user's profile is created.
   It is rare to use this lifetime outside of some metrics that are built in to Glean, such as `client_id`.
```

Comment:
The lifetime language here hasn't changed, but probably needs to be improved as it relates to the other discussion of lifetimes in the "simple value" section (which is where getting the lifetimes matters most, IMHO).

Reply:
Yes, good idea. What improvements did you have in mind? Any relevant example we might just embed in the docs?

Reply:
As I said above, I think any 'real-life' examples we could provide that illustrate our intentions with lifetimes would be useful.